docs/en/05_Contributing/04_Release_Process.md

```diff
@@ -12,13 +12,16 @@ and [silverstripe/recipe-cms](https://github.com/silverstripe/recipe-cms)).
 
 ## Release Planning
 
-Releases are planned by assigning issues to GitHub milestones,
```

Comment:
We don't really use Milestones like that anymore. We use them them indirectly through ZenHub, but I don't think an community member would have any hope of figuring out what's going to be in the next minor release by looking at the milestone.